qp_klp/Step.py

```diff
@@ -137,7 +137,7 @@ def update_sample_sheet(cls, sample_sheet_path, lane_number):
             sheet.write(f)
 
     @classmethod
-    def parse_prep_file(cls, prep_file_path):
+    def parse_prep_file(cls, prep_file_path, convert_to_dict=True):
```

Comment:
Just to be sure, is this really needed? I mean is there calls to parse_prep_file where convert_to_dict is True? I only see Falses here.

Reply:
Arguably the most important use of parse_prep_file():
https://github.com/charles-cowart/qp-knight-lab-processing/blob/37736e223f6f6f7ed10ec985974ad1ec09467df8/qp_klp/Step.py#L188